ui/page/file/view.jsx

```diff
@@ -74,26 +73,67 @@ class FilePage extends React.Component<Props> {
     markSubscriptionRead(channelUri, uri);
   }
 
+  renderFilePageLayout(uri, mode, cost) {
```

Comment:
these are essentially the 4 ways a page can be laid out:
- file pages with floating player (do not render the file, let FileRenderFloating fill area rendered FileRenderInitiator)
- unrenderable files - present as downloads, never use Initiator
- text modes - render Initiator but replaced with inline player once started + title on top
- everything else - same as above, but title below (images mainly, but technically everything else)